agent/ansible/roles/pbench-repo-install/templates/etc/yum.repos.d/pbench-test.repo.j2

```diff
@@ -0,0 +1,11 @@
+###########################################################################
+# External COPR repo
```

Comment:
Can we remove the notion of having a test vs production set of repos and just have one play book that works using a variable that can be changed for test vs production?